### PR TITLE
fix(create-new): Add login option to cloud beta template list and fix styles

### PR DESCRIPTION
**Pull request by tristandubbeld**
## What kind of change does this PR introduce?

Adds option to sign in via the cloud templates tab, as you need to be logged in to fork a cloud template:

image

Adds option to sign in via the error message:

image

I've also updated some CSS to align the headings in the different tab panels.

This is a first iteration to quickly improve UX. It makes it easier to find the login button in the create new modal. In the future we might remove this again when we have a better option.

- [x] Ready to be merged

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/7015/commits/416256ec0ec01017ac963e2f89c2e9690c1c10b0 is at https://pr7015.build.csb.dev/s/new.

File: packages/app/src/app/components/CreateSandbox/TemplateCategoryList.tsx [hunk 1]
```diff
@@ -42,14 +46,31 @@ export const TemplateCategoryList = ({
...
       </Stack>
+      {!hasLogIn && isCloudTemplateList ? (
+        <Stack direction="vertical" gap={4}>
+          <Text id="unauthenticated-label" css={{ color: '#999999' }} size={3}>
+            You need to be signed in to fork a cloud template.
+          </Text>
+          <Button
+            aria-describedby="unauthenticated-label"
+            css={{
+              width: '132px',
```
Comment (alexnm) [inline packages/app/src/app/components/CreateSandbox/TemplateCategoryList.tsx:65]: so precise :D

**Review by alexnm — APPROVED** [review]
Looks good

File: packages/app/src/app/components/CreateSandbox/TemplateCategoryList.tsx (hunk 1 above)
Comment (tristandubbeld) [reply packages/app/src/app/components/CreateSandbox/TemplateCategoryList.tsx:65]: heh, copied this from somewhere 😄

**Pull request merged**